corporate/lib/stripe.py

```diff
@@ -870,12 +870,8 @@ def get_card_update_session_data_for_upgrade(
             "stripe_session_id": stripe_session.id,
         }
 
-    def create_stripe_checkout_session(
-        self,
-        metadata: Dict[str, Any],
-        session_type: int,
-    ) -> stripe.checkout.Session:
-        # Create checkout sessions for adding and updating exiting cards.
+    def get_card_update_session_data(self) -> Dict[str, Any]:
```

Comment:
This name change may not be ideal for a function that mutates state; generally I think of `get_` functions as being read-only operations.

Reply:
Did this tweak before pushing https://github.com/zulip/zulip/pull/28026.